core/src/main/java/com/linecorp/armeria/common/AbstractContextAwareExecutorService.java

```diff
@@ -0,0 +1,107 @@
+package com.linecorp.armeria.common;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.stream.Collectors;
+
+import javax.annotation.Nullable;
+
+abstract class AbstractContextAwareExecutorService implements ExecutorService {
+    protected final ExecutorService executor;
+
+    AbstractContextAwareExecutorService(ExecutorService executor) {this.executor = executor;}
```

Comment:
This will probably fail Checkstyle tasks.

Reply:
I can see a build failure with 6 files with Checkstyle violations, how do I view the report?

Reply:
You can see the report by executing `./gradlew :core:checkstyle`.

Reply:
Or could be on GitHub Actions. https://github.com/line/armeria/runs/2453447519?check_suite_focus=true